h2o-genmodel/src/main/java/hex/genmodel/easy/BinaryColumnMapper.java

```diff
@@ -0,0 +1,39 @@
+package hex.genmodel.easy;
+
+import hex.genmodel.GenModel;
+import hex.genmodel.utils.MathUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BinaryColumnMapper {
+
+  private final GenModel _m;
+
+  public BinaryColumnMapper(GenModel m) {
+    _m = m;
+  }
+
+  public Map<String, Integer> create() {
+    String[] origNames = _m.getOrigNames();
+    String[][] origDomainValues = _m.getOrigDomainValues();
+    Map<String, Integer> columnMapping = new HashMap<>(origNames.length);
+    int pos = 0;
+    // non-categorical
+    for (int i = 0; i < _m.getOrigNumCols(); i++) {
+      if (origDomainValues[i] != null)
+        continue;
```

Comment:
the continue statement is confusing why not just use `if (origDomainValues[i]  == null) {}`

Reply:
fixed